packages/frontend/src/hooks/usePositions.ts

```diff
@@ -120,7 +120,7 @@ export const usePositions = () => {
         totalUSDSpent: bigZero,
         ethCollateralPnl: bigZero,
       },
-    [isWethToken0, swaps?.length],
+    [ethPrice.toString(), isWethToken0, swaps?.length],
```

Comment:
Do we need to add ethPriceMap to dep array as well?

Reply:
yh